zerver/actions/user_groups.py

```diff
@@ -180,8 +180,10 @@ def promote_new_full_members() -> None:
 
 def do_send_create_user_group_event(
     user_group: NamedUserGroup,
-    members: list[UserProfile],
-    direct_subgroups: Sequence[UserGroup] = [],
+    member_ids: list[int],
```

Comment:
Refactored this to just use IDs so that we can just fetch IDs using `values("id")` instead of fetching complete objects.